test/commands/console/index.test.js

```diff
@@ -71,8 +73,11 @@ describe('ConsoleCommand', () => {
           return null
         })
         command.printConsoleConfig()
-        expect(command.log).toHaveBeenCalledTimes(1)
-        expect(command.log).toHaveBeenCalledWith(noOrgErrorMessage)
+        expect(command.log).toHaveBeenCalledTimes(4)
+        expect(command.log).toHaveBeenCalledWith('You are currently in:')
+        expect(command.log).toHaveBeenCalledWith('1. Org: <no org selected>')
+        expect(command.log).toHaveBeenCalledWith('2. Project: THE_PROJECT')
```

Comment:
not sure this makes sense, how can you select a project without an org being selected? I get that this tests a permutation, but the config shouldn't ever get to this state (can the test be removed instead?)

Reply:
I agree, I will fix it along with #118